Source/Core/UICommon/CMakeLists.txt

```diff
@@ -9,13 +9,17 @@ set(SRCS
   VideoUtils.cpp
 )
 
+target_link_libraries(uicommon PUBLIC
+  common
+  cpp-optparse
+)
+
 if(USE_X11)
-  set(SRCS ${SRCS} X11Utils.cpp)
+  target_sources(uicommon PRIVATE X11Utils.cpp)
 endif()
 
-set(LIBS common cpp-optparse)
 if(LIBUSB_FOUND)
-  set(LIBS ${LIBS} ${LIBUSB_LIBRARIES})
+  target_link_libraries(uicommon PUBLIC ${LIBUSB_LIBRARIES})
```

Comment:
libusb isn't exposed publicly and is only used in the implementation, so could `PUBLIC` be `PRIVATE` instead here?